src/api/customer/customer-validator.ts

```diff
@@ -34,6 +34,18 @@ const createCustomerValidator = () => {
     });
 };
 
+const findByIdCustomerValidator = () => {
+    return celebrate({
+        [Segments.PARAMS]: Joi.object().keys({
+            customerUuid: Joi
+                .string()
+                .trim()
```

Comment:
Poderia ter sido mais restritivo e validar se o parâmetro é um UUID válido.

```suggestion
                .guid({ version: [ 'uuidv4'] })
```